src/Employer/Employer.Web/Orchestrators/Part1/TitleOrchestrator.cs

```diff
@@ -62,6 +64,11 @@ public TitleViewModel GetTitleViewModel()
 
             var vacancy = await _client.GetVacancyForEditAsync(vm.VacancyId.Value);
 
+            if (!vacancy.CanEdit)
```

Comment:
one for the longer term...

i think that enforcement of this type of rule will be beyond the scope of "current" the state of single entities.

we'll end up with decision making classes that, for a given entity, pull in other required dependencies (eg. data) and can answer questions like "can be extended", "can be closed", etc.

for now having a computed property is OK (perhaps better on the view model?) but i imagine this will disappear later on.

we'll also need to make such decisions outside of the web app too.